tests/jsdom.ts

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable */
+const { JSDOM } = require('jsdom');
+
+function copyProps(src: any, target: any) {
```

Comment:
Can you add some documentation here to point to what this test does?

Reply:
done!